rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/Rename.rsc

```diff
@@ -523,16 +537,19 @@ ProjectFiles preloadFiles(set[loc] workspaceFolders, loc cursorLoc) {
 }
 
 ProjectFiles allWorkspaceFiles(set[loc] workspaceFolders, str cursorName, bool(loc, str) containsName, PathConfig(loc) getPathConfig) {
-    return {
-        // If we do not find any occurrences of the name under the cursor in a module,
-        // we are not interested in loading the model, but we still want to inform the
-        // renaming framework about the existence of the file.
-        <folder, containsName(file, cursorName), file>
-        | folder <- workspaceFolders
-        , PathConfig pcfg := getPathConfig(folder)
-        , srcFolder <- pcfg.srcs
-        , file <- find(srcFolder, "rsc")
-    };
+    ProjectFiles fs = {};
```

Comment:
this change seems to be due to:

> Rewrite the workspace crawl to use less environments.

How come this is a concern? how much slower is it to write is as a comprehension?

Reply:
This is more of a general attempt to remove the use of backtracking generators, and turned out to be in the path of debugging this issue. I can not imagine this showing up in any real-use profile. I can revert if you prefer the comprehension. Let me know.

Reply:
well, as a reviewer, I look at a change and think: why is this a change that we need. So let's revert is, and keep the comprehension. 

(a variant of: no need for premature optimization)

Reply:
Ok, reverted!